lead-engineer-frontend.md

```diff
@@ -5,13 +5,33 @@
 
 _Victoria BC &middot; Full-Time_
 
-We're looking for an experienced frontend web developer to help architect and lead our customer facing applications.
+Sendwithus is a YC W2014 backed company looking for an amazing Lead Frontend Developer. We are a fun loving, hard working, and motivated team that loves to ship code and create amazing customer experiences. We want someone who can work with our team of developers and contribute to our lively team culture. We are a small start-up based in Victoria BC and San Francisco CA and we want that perfect person to grow and scale with us.
 
 <!-- more -->
 
-You'll be responsible for architecture and technical design of all user facing applications, including our primary UI (the sendwithus dashboard), landing pages, and other public resources.
+__What we do__ 
 
-This is not a content or design role; this is a high-impact, high-ownership technical opportunity. You'll be leading our frontend team and working closely with our product team to ensure everything runs smooth and looks great.
+Sendwithus is a powerful, easy to use dashboard and API that work together to help you design, test, and optimize 
+your transactional and marketing emails. Use our editor to make changes to your templates on the fly, set up A/B 
+tests, and see advanced downstream analytics - all without changing your code, or involving a developer.
+
+
+__What we offer__
+
+* Competitive salary + equity + benefits + vacation + parental leave
+* Be challenged and rewarded as you lead all aspects of front-end development
+* Rapid professional growth
+* Work with an amazing team of like minded professionals/ tech nerds
+* Take part in annual company retreat (Went to Whistler for 2015)
```

Comment:
What's in the parenthesis could be more exciting:
`(Whistler Party June 2015!)`